### PR TITLE
Merge Release 0.5.2 Changelog into master

### PR DESCRIPTION
**Pull request by ericholscher**
This is merging the changes from 0.5.2 back into `master`, so the changelog is up to date.

### CODE REVIEW

**Comment by ericholscher** [comment]
```
-> git diff --stat 0.5.1 rel-0.5.2
 docs/changelog.rst | 9 +++++++++
 package-lock.json  | 2 +-
 package.json       | 2 +-
 setup.py           | 1 +
 4 files changed, 12 insertions(+), 2 deletions(-)
```

**Review by agjohnson — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def run(self):
     },
     install_requires=[
         'sphinx'
+        'docutils<0.17', # https://github.com/sphinx-doc/sphinx/issues/9001
```
Comment (agjohnson) [inline setup.py:119]: Missed the preceding `,` in the list here. Requires got parsed as `sphinxdocutils`

**Comment by ericholscher** [comment]
This release is out: 

* https://pypi.org/project/sphinx-rtd-theme/0.5.2/
* https://github.com/readthedocs/sphinx_rtd_theme/releases/tag/0.5.2

**Comment by ericholscher** [comment]
This PR has been updated to now be the changes from 0.5.2 being merged back into `master` so we don't lose them. 👍

**Review by Blendify — APPROVED** [review]

**Pull request merged**